static/templates/browser/browser.hbs

```diff
@@ -490,7 +490,7 @@
             <a class = "name-icon">
               <div class = "item-name">{{item.name}}</div>
               {{#if item.compendium}}
-              <div class="icon" title="Compendium Item" style="background-image: url(systems/wfrp4e/ui/sidebar-button-compendium.webp)"></div>
+              <div class="icon" title="Compendium Item ({{item.compendium.metadata.label}}@{{item.originModule}})" style="background-image: url(systems/wfrp4e/ui/sidebar-button-compendium.webp)"></div>
```

Comment:
Not sure I like this format. `compendium.metadata.label` should include the module, right? LIke `Items (Up in Arms)`

Note that I will be converting all the Core compendium packs into one pack per type, so the `Spells`, `Talents`, etc. compendium packs will eventually be `Items (Core Rulebook)`

Reply:
> Not sure I like this format. `compendium.metadata.label` should include the module, right? LIke `Items (Up in Arms)`

That's only true for Compendiums you make. For example if I make a module with Compendium and Compendium is literally called "Items", then `compendium.metadata.label` will show `Items`. 

I made it thinking about 3rd party modules and people's own custom Compendiums as well.

I agree that format `Items@Forien's Armoury` is not perfect, but couldn't think of any better way to do it in tooltip. 


> Note that I will be converting all the Core compendium packs into one pack per type, so the `Spells`, `Talents`, etc. compendium packs will eventually be `Items (Core Rulebook)`

Please at least do folders inside :D

Reply:
I guess Module is not _that_ important if we assume all fan modules will have unique names of Compeniums then it shouldn't be an issue.

Reply:
> Please at least do folders inside :D

Of course! That's the point

But that is true, but instead of `Label@Module` let's just do `Label (Module)`

Reply:
Alright. Gonna go sleep now, will update the PR tomorrow

Reply:
Hm, that would do something like `Items (Forien Armoury) (Forien's Armoury)` though, so I'm not sure either.

Reply:
> Hm, that would do something like `Items (Forien Armoury) (Forien's Armoury)` though, so I'm not sure either.

I'm slowly shifting towards leaving just Label. Official modules will have module Included in the label, I can follow suit with my module and we can just assume others will as well.

Unofficial Grimoire prefixes all Compendiums with "UG" as well.